sample/kotlin/src/main/kotlin/com/datadog/android/sample/gdpr/GdprDialogFragment.kt

```diff
@@ -55,4 +63,10 @@ internal class GdprDialogFragment : DialogFragment() {
             TrackingConsent.NOT_GRANTED -> R.id.not_granted
         }
     }
+
+    @OptIn(ExperimentalRumApi::class)
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        GlobalRumMonitor.get().addViewLoadingTime()
```

Comment:
`onResume` will be called right after `onViewCreated` effectively discarding this call